src/viam/sdk/common/client_helper.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include <grpcpp/client_context.h>
+#include <viam/sdk/common/proto_type.hpp>
+#include <viam/sdk/common/utils.hpp>
+
+namespace viam {
+namespace sdk {
+
+template <typename ClientType, typename StubType, typename RequestType, typename ResponseType>
+class ClientHelper {
+    static void default_rsc_(RequestType&) {}
+    static void default_rhc_(const ResponseType&) {}
+    static void default_ehc_(const ::grpc::Status& status) {
+        throw std::runtime_error(status.error_message());
+    }
+
+   public:
+    using PFn = ::grpc::Status (StubType::*)(::grpc::ClientContext*,
```

Comment:
[q] I got the other abbreviations, but what does `PFn` stand for?

Reply:
Ah gotcha; no that seems fine. Could do `Method`, but I don't have strong opinion.